### PR TITLE
Add XP version bump to eap-version-bump.sh

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Comment by spyrkob** [comment]
That's going to fail if/when we go into CP10+ isn't it?

**Comment by xstefank** [comment]
@spyrkob yes, I don't think that is the issue since we will need to remove this with XP 2.0.0 nevertheless. And I expect that will be before 7.3.10.

**Comment by xstefank** [comment]
@spyrkob better safe than sorry :) , updated with your request.

**Comment by spyrkob** [comment]
nice :)

File: eap-version-bump.sh [hunk 1]
```diff
@@ -41,8 +41,12 @@ echo 'Done.'
 readonly PRODUCT_POM=${PRODUCT_POM:-'./pom.xml'}
 readonly PRODUCT_VERSION_TXT=${PRODUCT_VERSION_TXT:-'feature-pack/src/main/resources/content/version.txt'}
 readonly PRODUCT_VERSION="${NEXT_VERSION}.GA"
+readonly XP_PRODUCT_VERSION="1.0.$(($(echo $NEXT_VERSION | sed -e "s/.*\.\(.*\)$/\1/") - 1)).GA"
```
Comment (rpelisse) [inline eap-version-bump.sh:44]: @spyrkob a bit too many parenthesis in this one, it makes me nervous :)

What about moving this part into a readonly var: $(echo $NEXT_VERSION | sed -e "s/.*\.\(.*\)$/\1/") - 1)    (if nothing else, for visibility purpose)

Comment (xstefank) [reply eap-version-bump.sh:44]: surprise @rpelisse! Who made the PR!!! :D will do.

**Comment by spyrkob** [comment]
@rpelisse can we merge this one?

File: eap-version-bump.sh [hunk 2]
```diff
@@ -41,8 +41,13 @@ echo 'Done.'
 readonly PRODUCT_POM=${PRODUCT_POM:-'./pom.xml'}
 readonly PRODUCT_VERSION_TXT=${PRODUCT_VERSION_TXT:-'feature-pack/src/main/resources/content/version.txt'}
 readonly PRODUCT_VERSION="${NEXT_VERSION}.GA"
+readonly XP_MINOR_VERSION=$(echo $NEXT_VERSION | sed -e "s/.*\.\(.*\)$/\1/")
+readonly XP_PRODUCT_VERSION="1.0.$(($XP_MINOR_VERSION - 1)).GA"
```
Comment (rpelisse) [inline eap-version-bump.sh:45]: I'm not sure what the meaning of $(( )) ? What does it do (compare to $() ) ? Oh I guess it allows to do math operation, right? I would use expr for this it is more explicit:

$( expr "${XP_MINOR_VERSION}" - 1 ) 

Also I would "protect" $XP_MINOR_VERSION => ${XP_MINOR_VERSION}

**Review by rpelisse — CHANGES_REQUESTED** [review]

**Comment by xstefank** [comment]
@rpelisse updated.

**Pull request merged**